pwndbg/commands/heap.py

```diff
@@ -96,6 +97,24 @@ def arena(addr=None):
 
     print(main_arena)
 
+
+@pwndbg.commands.ParsedCommand
+@pwndbg.commands.OnlyWhenRunning
+def arenas():
+
+    heap  = pwndbg.heap.current
+    addr  = None
+    arena = heap.get_arena(addr)
+    main_arena_addr = int(arena.address)
+    while addr != main_arena_addr:
+        
+        low,top = heap.get_bounds(addr)
```

Comment:
Ah, I see now that `get_bounds` returns a `tuple` object.  It might be worth updating this code to return `pwndbg.memory.Page` object, which describes a region of memory (the name `Page` is a misnomer, and should also probably be updated!)